File: examples/java/src/cucumber/demo/Hello.java
package cucumber.demo;

public class Hello {
    public String greet(String who, String from) {
        return "Hi, " + who + ". I'm " + from;
    }
    
    public boolean isFriend(String who) {
        return true;
    }
    
    public String getPhoneNumber(String who) {
        throw new RuntimeException("NOPE");
        //return "98219458";
    }
}

File: examples/cs/src/Hello.cs
using System;

namespace Cucumber.Demo {
	public class Hello {
    public string Greet(string who, string from) {
        return "Hi, " + who + ". I'm " + from;
    }
   
    public bool IsFriend(string who) {
        return true;
    }
   
    public string GetPhoneNumber(string who) {
       	return "99999";
//throw new Exception("My phone is secret!");
    }
	}
}